articles/operator-nexus/howto-configure-cluster.md

```diff
@@ -167,6 +167,19 @@ provided through the Cluster's rack definition. Based on the results of these ch
 and any user skipped machines, a determination is done on whether sufficient nodes
 passed and/or are available to meet the thresholds necessary for deployment to continue.
 
+> [!IMPORTANT]
+> As of NC 3.4.0, the hardware validation process will write the results to the specified `analyticsWorkspaceId` at Cluster Creation.
+> Additionally, the provided Service Principal in the Cluster object is used for authentication against the Log Analytics Workspace Data Collection API.
+> This capability is only visible during a new deployment (Green Field); existing cluster will not have the logs available retroactively.
```

Comment:
It's fine this is GF only but since existing clusters will not have this retroactively, what will existing clusters have? I don't think this should be silent.